Thu Sep 15 14:12:43 2022 - INFO  - Start to prepare the test environment.
Failed to start mariadb.service: Unit mariadb.service not found.
Unit mariadb.service could not be found.
Thu Sep 15 14:13:11 2022 - INFO  - Start to restore the test environment.
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: 'Error: \n Problem: package mariadb-server-4:10.5.16-1.oe2209.riscv64 requires mariadb(riscv-64), but none of the providers can be installed\n  - conflicting requests\n  - nothing provides mariadb-connector-c >= 3.0 needed by mariadb-4:10.5.16-1.oe2209.riscv64'
Thu Sep 15 14:13:15 2022 - INFO  - End to restore the test environment.
